packages/core/package.json

```
@@ -21,7 +21,7 @@
   "exports": {
     ".": {
       "import": "./dist/index.js",
-      "require": "./dist/index.cjs"
+      "require": "./dist/index.js"
     }
```

I was looking at npm docs earlier today and realized if we're referencing the same file, we can simplify this

```suggestion
    ".": "./dist/index.js"
```

---

Nice one! Updated